pom.xml

```
@@ -1247,6 +1247,11 @@
       <!-- Let's mark as optional as it is not strictly needed -->
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.14</version>
```

1.15 seems to be the most recent: https://repo1.maven.org/maven2/commons-codec/commons-codec/1.15/

```suggestion
      <version>1.15</version>
```